build.cmd

```diff
@@ -1,15 +1,8 @@
 @echo off
+setlocal
 
-if "%~1"=="-h" goto help
-if "%~1"=="-help" goto help
-if "%~1"=="-?" goto help
-if "%~1"=="/?" goto help
+set _args="%*"
+if "%~1"=="-?" set _args="-help"
```

Comment:
this works without `EnableDelayedExpansion`?

Reply:
IIRC "EnableDelayedExpansion" only affects the !var! syntax within conditional blocks.

Reply:
Right, I'm asking if `%_args%` in L7 ever resolves to `-help`.

Reply:
I was assuming that for this to work we would need the following code:

```
L2: setlocal EnableDelayedExpansion
L7: !_args!
```

Reply:
Since the usage is outside of the if block I believe we don't need EnableDelayedExpansion. I tested it and it worked just fine.

Reply:
You are right:

> For simple commands this will make no noticable difference, but with loop commands like FOR, compound or bracketed expressions delayed expansion will allow you to always see the current value of the variable.

https://ss64.com/nt/delayedexpansion.html